robottelo/host_helpers/satellite_mixins.py

```diff
@@ -201,7 +201,7 @@ def is_sca_mode_enabled(self, org_id):
         """
         return self.api.Organization(id=org_id).read().simple_content_access
 
-    def publish_content_view(self, org, repo_list):
+    def publish_content_view(self, org, repo_list, name):
```

Comment:
```suggestion
    def publish_content_view(self, org, repo_list, name=gen_string('alpha')):
```

@JacobCallahan, this is generally used in `tests/upgrades`. Without the added positional argument.
```
>       content_view = target_sat.publish_content_view(function_org, repolist)
E       TypeError: ContentInfo.publish_content_view() missing 1 required positional argument: 'name'
```